types/oscal/profile/profile.go

```diff
@@ -118,11 +118,6 @@ type Combine struct {
 // elements or attributes.
 type AsIs string
 
-func (a *AsIs) UnmarshalJSON(b []byte) error {
```

Comment:
this may cause errors in unmarshalling profiles

Reply:
This was re-generated from the profile metaschema which, as we've discussed, has a bug per https://github.com/usnistgov/OSCAL/issues/287. I'll revert these changes.